x-pack/plugins/security_solution/public/detection_engine/rule_gaps/components/manual_rule_run/index.tsx

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import {
+  EuiBetaBadge,
+  EuiConfirmModal,
+  EuiDatePicker,
+  EuiDatePickerRange,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiForm,
+  EuiFormRow,
+  EuiHorizontalRule,
+  EuiSpacer,
+  useGeneratedHtmlId,
+} from '@elastic/eui';
+import moment from 'moment';
+import React, { useCallback, useMemo, useState } from 'react';
+import { TECHNICAL_PREVIEW, TECHNICAL_PREVIEW_TOOLTIP } from '../../../../common/translations';
+
+import * as i18n from './translations';
+
+interface ManualRuleRunModalProps {
+  onCancel: () => void;
+  onConfirm: (timeRange: { startDate: moment.Moment; endDate: moment.Moment }) => void;
+}
+
+const ManualRuleRunModalComponent = ({ onCancel, onConfirm }: ManualRuleRunModalProps) => {
+  const modalTitleId = useGeneratedHtmlId();
+
+  const now = useMemo(() => moment(), []);
```

Comment:
useMemo feels redundant here, as on every render now should change and reflect current time instead of memoized

Reply:
done